huxley/core/tests/models/test_models.py

```diff
@@ -99,9 +101,11 @@ def test_update_fees(self):
             advanced_delegates=a,
         )
 
+        registration_fee = Conference.get_conference().registration_fee
+        delegate_fee = Conference.get_conference().delegate_fee
```

Comment:
local variable. (I don't mean to be nitty, but I don't want to rely on the ORM's caching behavior to that degree).